.gitlab-ci.yml

```
@@ -1765,6 +1765,12 @@ deploy_windows_testing-a7:
   script:
     - bash -l tasks/run-test-kitchen.sh upgrade7-test $AGENT_MAJOR_VERSION
 
+.kitchen_test_runtime_security:
+  script:
+    - inv -e system-probe.build --bundle-ebpf --incremental-build -no-with-bcc
```

I think there's a missing dash here, I'm getting an error when doing:
```
$ inv system-probe.build -no-with-bcc
No idea what '-t' is!
```

```suggestion
    - inv -e system-probe.build --bundle-ebpf --incremental-build --no-with-bcc
```

---

Indeed. Fixed